OpenEdXMobile/res/values/styles.xml

```diff
@@ -214,6 +215,14 @@
         <item name="iconColor">@color/white</item>
     </style>
 
+    <style name="video_player_advance_control_time">
```

Comment:
IMO better to add `_text` at the end that describes the style is used in text views.